data/free worlds reconciliation.txt

```diff
@@ -1554,6 +1560,643 @@ mission "FW Pug 3C"
+			choice
+				`	"An alien species called the Korath."`
+					goto true
+				`	"A species to the north of Syndicate space called the Korath."`
+					goto true
+				`	"The Quarg."`
+					goto "quarg drive"
+
+			label "quarg drive"
+			`	Instantly uproar brakes out. The speaker spends several moments shouting at everyone to be quite before they can continue.`
+			`	One of the Dirt Belt representative gets up, "We had heard rumours but we never believed that it could be done."`
+			`	A representative from Shangri-La stands saying "Another species with Jump Drives, called the Korath, has been raiding our planet, maybe Captain <last> has got Karath tech instead."`
+			`	Now a Paradise planet representative, "Implying that you have defeated the Quarg is just an attempt to scare us into compliance!" This raises a "Hear, Hear." from the other Paradise planet reps.`
+				goto last
+
+			label "refuse 2"
+			`	Alondo says, quietly "Captain <last>, if we are to get anywhere we need them to believe us, and if you refuse to tell them where you got your Jump Drive we won't get anywhere."`
+			choice
+				`	"An alien species called the Korath."`
+				`	"A species to the north of Syndicate space called the Korath."`
+
+			label true
+			`	"It's true." A representative from Shangri-La has stood up. "My system has been raided by the Korath. They have powerful ships but they can be defeated and they can be disabled. It's not inconceivable that Captain <last> took one of their Jump Drives."`
+			`	Murmurs sweep round the chamber before one of the Paradise planet representatives says, "You now have two aliens in your story, this is getting less believable by the minute!" This raises a "Hear, Hear." from the other Paradise planet reps."`
+
+			label last
+			`	Alondo sighs before stepping forward and raising his voice so as to be heard, "The Pug are occupying systems that used to be only a single jump from here. We need to act now, as a united species, as one Human race. Now is not the time to fight each other, we need the Navy and you need our Jump Drive. Every moment we are not acting is another moment longer our worlds are subjugated."`
+			`	One of the Midgard representatives stands. "The Navy has sensors that could confirm that Captain <last>'s ship arrived using a Jump Drive." The Speaker asks you to leave while Parliament hears what the Navy knows and decides what should be done.`
+			`	Outside Alondo looks stressed, "Even when they have all the facts they still can't decide things quickly. Well I suppose our Senate is not much better."`
+			choice
+				`	"Politicians always decide things slowly, and these ones are running scared."`
+					goto scared
+				`	"It can be tiring sometimes but we can't run everything."`
+					goto run
+
+			label scared
+			`	At that moment a Navy admiral hurries past looking stressed. "Your right there," says Alondo, "must be all that public exposure they get making them far to careful."`
```

Comment:
This line: Your - You're*
1791: Karath - Korath*, planet - system(?)*
1783: North - East/North-East(?)*